src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -0,0 +1,156 @@
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.notes ? 
+                                        <label className="label-component details">Notes</label>
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.emergencypickup ? 
+                                        <label className="label-component details">Emergency Pickup</label>
+                                        :
+                                        <span />
+                                    }
+                                </div>
+                                
+                                <div className="editing-child-2">
+                                    <input name="name" type="text" className="form-input" defaultValue={this.props.account.name} /><br /><br />
+                                    <div className="scs-spacing" />
+                                    <input name="street1" type="text" className="form-input" defaultValue={this.props.account.address.street1} /><br /><br />
+                                    <input name="street2" type="text" className="form-input" defaultValue={this.props.account.address.street2} /><br /><br />
+                                    <input name="city" type="text" className="form-input" defaultValue={this.props.account.address.city} /><br /><br />
+                                    <input name="state" type="text" className="form-input" defaultValue={this.props.account.address.state} /><br /><br />
+                                    <input name="zip" type="text" className="form-input" defaultValue={this.props.account.address.zipcode} /><br /><br />
+                                    <input name="phone" type="tel" className="form-input" defaultValue={this.props.account.organizationPhone} /><br /><br />
+                                    {this.props.account.numVolunteers ? 
+                                        <input name="nv" type="text" className="form-input" defaultValue={this.props.account.numVolunteers} />
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.notes ? 
+                                        <input name="notes" type="text" className="form-input" defaultValue={this.props.account.notes} />
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.emergencypickup ? 
+                                        <input name="ep" type="text" className="form-input" defaultValue={this.props.account.emergencypickup} />
```

Comment:
this shouldn't be a text input? This should be a slider or checkbox or something that's for turning it on or off

Reply:
Fixed